bridges/bin/runtime-common/src/messages_benchmarking.rs

```diff
@@ -290,19 +290,19 @@ where
 /// Returns callback which generates `BridgeMessage` from Polkadot XCM builder based on
 /// `expected_message_size` for benchmark.
 pub fn generate_xcm_builder_bridge_message_sample(
-	destination: InteriorMultiLocation,
+	destination: InteriorLocation,
 ) -> impl Fn(usize) -> MessagePayload {
 	move |expected_message_size| -> MessagePayload {
 		// For XCM bridge hubs, it is the message that
 		// will be pushed further to some XCM queue (XCMP/UMP)
-		let location = xcm::VersionedInteriorMultiLocation::V3(destination);
+		let location = xcm::VersionedInteriorLocation::V4(destination.clone());
 		let location_encoded_size = location.encoded_size();
 
 		// we don't need to be super-precise with `expected_size` here
 		let xcm_size = expected_message_size.saturating_sub(location_encoded_size);
 		let xcm_data_size = xcm_size.saturating_sub(
 			// minus empty instruction size
-			xcm::v3::Instruction::<()>::ExpectPallet {
+			xcm::v4::Instruction::<()>::ExpectPallet {
```

Comment:
You're already importing the latest version, so you can just do this:

```suggestion
			Instruction::<()>::ExpectPallet {
```